### DISCUSSION TITLE
Do you need to pay for an enterprise to delete a file with sensitive data?

### DISCUSSION THREAD

**Discussion by aaronfactbid** [community/community#41881] [Enterprise]
### Select Topic Area

Question

### Body

I accidentally committed a SQL file which contained our SMTP key.  I immediately removed it with git reset -hard and git push --force.  Now the commit no longer appears, but it's still accessible from the original URL with the commit hash, and when I open it I get the "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."  I cannot delete the commit, nor the file.  So I followed the instructions in the github help page "Removing sensitive data from a repository" and used git-filter-repo.  But it's still on github.com.  The next step says "Contact GitHub Support, asking them to remove cached views"...  But I have a FOSS project and the link to create a ticket says I need an enterprise account.  So is there really a dead-end and I have to just delete the entire repository, losing all history and years of work, because there's no way to permanently delete something on github.com myself without buying a support package?  I've already deleted the old key, but still my SMTP provider (SendGrid) will not reactivate my account until the old key is no longer on github.com.

**Comment by vibhormehta** [comment]
If you want, you can make the repository private. Your key would be more secure if you do that.
